cdc/sink/mysql.go

```diff
@@ -364,6 +354,16 @@ func (s *mysqlSink) execDDL(ctx context.Context, ddl *model.DDLEvent) error {
 	return nil
 }
 
+func needSwitchDB(ddl *model.DDLEvent) bool {
```

Comment:
to reduce indent level, we can

if len == 0 {
return false
}
if ddl.Type == timodel.ActionCreateSchema || ddl.Type == timodel.ActionDropSchema {
return false
}
return true

also you can use `switch` to replace last `if`

Reply:
Done. Thanks!